droidlet/tools/artifact_scripts/tracked_checksums/datasets.txt

```diff
@@ -1 +1 @@
-7421f7de5cabac42fb5a78651d0855fc44516cae
+f0c28e8acad0a327d2431d6acdc7700dd9b5a89c
```

Comment:
Is this the same as what's in #1263 ?

Reply:
Yeah, it is. Since i am not sure which PR will be merged first, just in case, I update datasets.txt in both PRs.